pkg/ioutil/ioutil.go

```diff
@@ -20,8 +20,13 @@ package ioutil
 
 import (
 	"io"
+	"os"
+
+	humanize "github.com/dustin/go-humanize"
 )
 
+var DefaultBufferSize = humanize.MiByte
```

Comment:
should this be const? - can you add a comment..

Reply:
Sure, I can make that constant too as this util func is used only by us.